infra/k8s/sidecar.yaml

```diff
@@ -36,7 +36,8 @@ spec:
           ip route || true
       containers:
       - name: testground-sidecar
-        image: ipfs/testground:edge
+        #image: ipfs/testground:edge
```

Comment:
I'm afraid we have introduced a regression in the `sidecar` - a commit from a few days ago is much more stable for me, otherwise it seems like `sidecar` is either losing connection to `docker.sock` or deadlocking (we have an issue for that). Not sure what is going on, and I haven't had time to investigate further just yet.

Reply:
Can we instead depend on [the past tag of the `ipfs/testground` image](https://hub.docker.com/r/ipfs/testground/builds) that was well-behaved?